samples/Playground/Playground.Shared/AppHost.cs

```diff
@@ -0,0 +1,190 @@
+﻿namespace Playground;
+
+internal static class AppHost
```

Comment:
Let's avoid this change and more this back into App.xaml.host.cs

Reply:
it needs to be in static class as it's an extension

Reply:
but we're moving this back into app.xaml.cs anyhow, right?